models/models_test.go

```diff
@@ -68,6 +68,9 @@ func TestAdminTemplateModel(t *testing.T) {
 		panic(unmarshallError)
 	}
 
+	expectedFallbackVersion := "${env://VAMP_PERSISTENCE_KEY_VALUE_STORE_VAULT_FALLBACK_KV_VERSION}"
```

Comment:
This is unnecessary. The test checks that all fields are the same in the resulting file.